packages/manager/src/features/Linodes/LinodesDetail/LinodeResize/LinodeResize.tsx

```diff
@@ -313,6 +317,12 @@ const getError = (error: APIError[] | null) => {
       </Typography>
     );
   }
+  if (
+    typeof errorText === 'string' &&
+    errorText.match(/Additional verification/i)
+  ) {
+    return <VerificationError title="Verification Request" />;
+  }
```

Comment:
I think we can delete this because the Axios interceptor will automatically do this.

```suggestion
  if (
    typeof errorText === 'string' &&
    errorText.match(/Additional verification/i)
  ) {
    return <VerificationError title="Verification Request" />;
  }
```

Reply:
Good catch!